website/docs/Installation/windows-install.md

```diff
@@ -3,69 +3,44 @@ sidebar_position: 4
 ---
 
 # Windows
-## 1. Installing Podman Desktop on Windows
 
-In order to install the latest Podman Desktop application for Windows, visit the [Downloads](/downloads/windows) section of this website to download the .exe file.
+This page contains information regarding installation of Podman Desktop on Windows.
 
-Simply download the file from the [Downloads](/downloads/windows) section and open it in your Desktop to install Podman Desktop.
+:::infoPrerequisite
+Podman Desktop on Windows requires the following:
+1. [Podman Engine](https://docs.podman.io/en/latest/index.html)
```

Comment:
is it a pre-requisite? I don't think so, as we guide the user into installing podman engine, if it is not yet installed.

Reply:
updated the banner

Reply:
updated here: https://github.com/containers/podman-desktop/pull/534/commits/ed0de667700877ba44d3fe5662dc596dd44ed000